upup/models/cloudup/resources/addons/networking.amazon-vpc-routed-eni/k8s-1.7.yaml.template

```diff
@@ -1,3 +1,44 @@
+# Vendored from https://github.com/aws/amazon-vpc-cni-k8s/blob/release-1.0/config/v1.0/aws-k8s-cni.yaml
+
+apiVersion: rbac.authorization.k8s.io/v1beta1
```

Comment:
Eventually we should probably create a second version of this manifest for 1.8+ with `rbac.authorization.k8s.io/v1`

Reply:
Sounds good. I'll add it to my list.